script/utils/CurrentConfigCheck.s.sol

```diff
@@ -0,0 +1,570 @@
+        assertEq(strategyFactory.owner(), operationsMultisig,
+            "strategyFactory.owner() != operationsMultisig");
+        assertEq(avsDirectory.owner(), executorMultisig,
+            "avsDirectory.owner() != executorMultisig");
+        assertEq(rewardsCoordinator.owner(), operationsMultisig,
+            "rewardsCoordinator.owner() != operationsMultisig");
+        assertEq(eigenLayerPauserReg.unpauser(), executorMultisig,
+            "eigenLayerPauserReg.unpauser() != operationsMultisig");
+        require(eigenLayerPauserReg.isPauser(operationsMultisig),
+            "operationsMultisig does not have pausing permissions");
+        require(eigenLayerPauserReg.isPauser(executorMultisig),
+            "executorMultisig does not have pausing permissions");
+        require(eigenLayerPauserReg.isPauser(pauserMultisig),
+            "pauserMultisig does not have pausing permissions");
+
+        (bool success, bytes memory returndata) = timelock.staticcall(abi.encodeWithSignature("admin()"));
+        require(success, "call to timelock.admin() failed");
+        address timelockAdmin = abi.decode(returndata, (address));
+        assertEq(timelockAdmin, operationsMultisig,
+            "timelockAdmin != operationsMultisig");
+
+        (success, returndata) = executorMultisig.staticcall(abi.encodeWithSignature("getOwners()"));
+        require(success, "call to executorMultisig.getOwners() failed");
+        address[] memory executorMultisigOwners = abi.decode(returndata, (address[]));
+        require(executorMultisigOwners.length == 2,
+            "executorMultisig owners wrong length");
+        bool timelockInOwners;
+        bool communityMultisigInOwners;
+        for (uint256 i = 0; i < 2; ++i) {
+            if (executorMultisigOwners[i] == timelock) {
+                timelockInOwners = true;
+            }
+            if (executorMultisigOwners[i] == communityMultisig) {
+                communityMultisigInOwners = true;
+            }
+        }
+        require(timelockInOwners, "timelock not in executorMultisig owners");
+        require(communityMultisigInOwners, "communityMultisig not in executorMultisig owners");
+
+        require(eigenTokenProxyAdmin != beigenTokenProxyAdmin,
+            "tokens must have different proxy admins to allow different timelock controllers");
+        require(eigenTokenTimelockController != beigenTokenTimelockController,
+            "tokens must have different timelock controllers");
+
+        // note that proxy admin owners are different but _token_ owners are the same
+        assertEq(Ownable(address(EIGEN)).owner(), address(eigenTokenTimelockController),
+            "EIGEN.owner() != eigenTokenTimelockController");
+        assertEq(Ownable(address(bEIGEN)).owner(), address(eigenTokenTimelockController),
+            "bEIGEN.owner() != eigenTokenTimelockController");
+        assertEq(eigenTokenProxyAdmin.owner(), address(eigenTokenTimelockController),
```

Comment:
this conflicts with the `script/deploy/holesky/Eigen_Token_Deploy.s.sol` where the owner is the ops multisig

Reply:
We could potentially just delete that token deploy script -- it feels like it should become part of the "deploy from scratch" script going forwards, but I think this may not be possible right now due to not being able to compile all contracts at once (conflicting OZ versions, in particular)?